BUILD.md

```diff
@@ -235,40 +254,7 @@ conan profile update env.CC=<path> default
 conan profile update env.CXX=<path> default
 ```
 
-
-## How to build and test
-
-Let's start with a couple of examples of common workflows.
```

Comment:
Why are these examples being removed?

Reply:
@intelliot They are rewritten per-os instead of intermixed

Reply:
post-rewrite, where are the instructions for Linux or MacOS?